Formula/docbook.rb

```diff
@@ -76,20 +76,29 @@ def install
     end
   end
 
+  def xmlcatalog
+    on_macos do
+      return "xmlcatalog"
+    end
+    on_linux do
+      return "#{Formula["libxml2"].opt_bin}/xmlcatalog"
+    end
+  end
+
   def post_install
     etc_catalog = etc/"xml/catalog"
```

Comment:
If it's only used in the `post_install` it should be a variable, not a method, no?